extension.neon

```diff
@@ -1,4 +1,6 @@
 parameters:
+	phpunit:
+		convertUnionToIntersectionType: true
 	additionalConstructors:
 		- PHPUnit\Framework\TestCase::setUp
 	earlyTerminatingMethodCalls:
```

Comment:
@ondrejmirtes I can create a PR to move these properties to `phpunit` structure if you want.

Reply:
This is a setting from core PHPStan that this extension adds to.